README.md

```diff
@@ -161,6 +161,12 @@ necessary association when their parent record is deleted. With Active Record,
 we can use the `dependent: :destroy` option to automatically remove associated
 records when the parent record is deleted.
 
+## Check For Understanding
+
+Before you move on, make sure you can answer the following question:
+
+1. What is the process for setting up the `dependent: :destroy` option?
```

Comment:
Suggestion:

1. In what situations would you need to use the `dependent: :destroy` option?